diff --git a/cmd/smbplugin/Dockerfile b/cmd/smbplugin/Dockerfile
index 4c4f2f049..94294291d 100644
--- a/cmd/smbplugin/Dockerfile
+++ b/cmd/smbplugin/Dockerfile
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM registry.k8s.io/build-image/debian-base:bookworm-v1.0.7
+FROM opvolger/debian-base:trixie-v1.0.0
 
 RUN apt update && apt upgrade -y && apt-mark unhold libcap2 && clean-install ca-certificates cifs-utils util-linux e2fsprogs mount udev xfsprogs
 
